src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ReportGenerator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String FIELDS_NAMES = "type,fruit,quantity\n";
+    private final List<FruitTransaction> transactions;
```

Comment:
The `transactions` field is declared but never used in this class. If it's not necessary, consider removing it to keep your class simple and focused on a single task.